pkg/gateway/model_build_targetgroup.go

```diff
@@ -87,7 +87,7 @@ func (b *SvcExportTargetGroupBuilder) Build(
 	return task.stack, nil
 }
 
-func (b *SvcExportTargetGroupBuilder) BuildTargetGroup(ctx context.Context, svcExport *anv1alpha1.ServiceExport) (*model.TargetGroup, error) {
+func (b *SvcExportTargetGroupBuilder) BuildTargetGroups(ctx context.Context, svcExport *anv1alpha1.ServiceExport) (*model.TargetGroup, *model.TargetGroup, error) {
```

Comment:
Since we are not following mcs spec anymore for import/export there is no constraint to create both HTTP/gRPC TG's at the same time. We can extend CRD with type of service and build only one TG that we need.

Reply:
@solmonk